File: birthday/submissions/accepted/birthday_loop.cpp
#include <iostream>
using namespace std;
typedef long long ll;

ll solve(ll N) {
    ll numDays = 0;
    for (int i = 1; i < N; i++){
        numDays += i * i;
    }
    return numDays + 3;
}

int main() {
    int T;
    cin >> T;
    for (int i = 0; i < T; i++) {
        ll N;
        cin >> N;
        cout << solve(N) << '\n';
    }
}

File: birthday/submissions/accepted/birthday_loop.py
def solve(N: int) -> int:
    numDays = 0
    for i in range(1, N):
        numDays += i * i
    return numDays + 3

def main():
    T = int(input())
    for _ in range(T):
        N = int(input())
        print(solve(N))

if __name__ == '__main__':
    main()
